repos/c2sm/packages/libxml2/package.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.pkg.builtin.libxml2 import Libxml2 as SpackLibxml2
+
+
+class Libxml2(SpackLibxml2):
+    conflicts('%nvhpc @21.1:')
```

Comment:
Are you sure about that? Because https://github.com/spack/spack/blob/v0.20.1/var/spack/repos/builtin/packages/libxml2/package.py#L91-L102 is supposed to fix this conflict.

Reply:
No I'm not sure... this is very much work in progress.

Reply:
this got added accidentally during the merge. Removing it now